Datasets/Sentinel_1.ipynb

```diff
@@ -0,0 +1,655 @@
+    "\n",
+    "The [Sentinel-1](https://en.wikipedia.org/wiki/Sentinel-1) mission is composed of a constellation of two Synthetic Aperture Radar (SAR) satellites, Sentinel-1A and Sentinel-1B, sharing the same orbital plane. Sentinel-1 provides dual polarisation capability, very short revisit times, and rapid product delivery.\n",
+    "\n",
+    "SAR data has the advantage of operating at wavelengths not impeded by cloud cover or a lack of illumination and can acquire data over a site during the day or night, and under most weather conditions. The Sentinel-1 mission is the European Radar Observatory for the Copernicus joint initiative of the European Commission (EC) and the European Space Agency (ESA) that can offer reliable and repeated wide area monitoring with its SAR instrument. \n",
+    "\n",
+    "Sentinel-1 products will benefit users interested in agriculture, forestry and land cover classification. The rapid data dissemination and short revisit cycles of Sentinel-1 together with its interferometric capabilities will also benefit emergency response users, such as the United Nations [International Charter on Space and Major Disasters](https://disasterscharter.org/web/guest/home;jsessionid=83C5354659FD4EFBE0A8D388BBA5B56F.jvm1), in emergency situations such as floods, earthquakes, volcanic eruptions and landslides.\n",
+    "\n",
+    "Sentinel-1A was launched on 3 April 2014 and Sentinel-1B followed on 25 April 2016. Two other spacecraft (Sentinel-1C and Sentinel-1D) are planned to replace the first two satellites at the end of their operational lifespan. For more information on the Sentinel-1 platforms and applications, check out the [European Space Agency](https://sentinel.esa.int/web/sentinel/missions/sentinel-1) website.\n",
+    "\n",
+    "The Normalized Radar Backscatter product needs Radiometric Terrain Correction (RTC) that is called gamma0 (γ0) backscatter. In this product the systematic contamination that would otherwise still be present in sets of data acquired with multiple geometries will be mitigated.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Modes, Swath Widths and Resolution\n",
+    "The C-SAR instrument on Sentinel-1 supports operation in dual polarisation (HH+HV, VV+VH) that is useful for land cover classification and sea-ice applications. Sentinel-1 operates in four exclusive acquisition modes:\n",
+    "* Strip Map Mode: 80 km swath, 5 x 5 m spatial resolution\n",
+    "* Interferometric Wide Swath: 250 km swath, 5 x 20 m spatial resolution\n",
+    "* Extra-Wide Swath Mode: 400 km swath, 20 x 40 m spatial resolution\n",
+    "* Wave-Mode: 20 x 20 km, 5 x 5 m spatial resolution\n",
+    "\n",
+    "<img align='center'\n",
+    "     src='../Supplementary_data/Sentinel1AB/Sentinel1_Modes.jfif'>\n",
+    "\n",
+    "\n",
+    "---\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Description\n",
+    "\n",
```

Comment:
Please remove all the information around loading the data and analysis. I am planning to update them soon.